hoprd/hoprd/example_cfg.yaml

```diff
@@ -0,0 +1,309 @@
+hopr:
```

Comment:
A good link for Hammad would be this file.

Reply:
Will send it to him once merged.